src/poliastro/twobody/orbit.py

```diff
@@ -235,6 +237,24 @@ def from_body_ephem(cls, body, epoch=None):
 
         return ss
 
+    @classmethod
+    def from_horizons(cls, name, epoch=None, plane=Planes.EARTH_EQUATOR):
+        if not epoch:
+            epoch = time.Time.now()
+        if plane == Planes.EARTH_EQUATOR:
+            refplane = "earth"
+        elif plane == Planes.EARTH_ECLIPTIC:
+            refplane = "ecliptic"
+        obj = Horizons(id=name, epochs=epoch.jd).elements(refplane=refplane)
+        a = obj["a"][0] * u.au
+        ecc = obj["e"][0] * u.one
+        inc = obj["incl"][0] * u.deg
+        raan = obj["Omega"][0] * u.deg
+        argp = obj["w"][0] * u.deg
+        nu = obj["nu"][0] * u.deg
+        ss = classical.ClassicalState(Sun, a * (1 - ecc ** 2), ecc, inc, raan, argp, nu)
```

Comment:
`*State` classes should never be used. I will make them private and remove them from the documentation.

Reply:
No, you should use `Orbit.from_classical`

Reply:
I am sorry for using that! I didn't know that I must not  use `*State`

Reply:
Oh! Yes, obviously. I am sorry again!